core/src/main/scala/tamer/Tamer.scala

```diff
@@ -341,10 +355,10 @@ object Tamer {
         .make(consumerSettings)
         .mapError(TamerError("Could not make state consumer", _))
       val stateProducer = Producer
-        .make(producerSettings, stateKeySerde.serializer, serdes.stateSerde)
+        .make(producerSettings)
         .mapError(TamerError("Could not make state producer", _))
       val valueProducer = Producer
-        .make(producerSettings, serdes.keySerializer, serdes.valueSerializer)
+        .make(producerSettings)
         .mapError(TamerError("Could not make value producer", _))
```

Comment:
I wonder if we should just now use one producer

Reply:
Yeah I think it fits with the purpose of the PR, I'll do it

Reply:
Great!! Probably also worth calling them just `consumer` and `producer` at this stage :)